src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1469,21 +1459,16 @@ protected override void ProcessRecord()
                                     HttpResponseException httpEx = new(message, response);
                                     ErrorRecord er = new(httpEx, "WebCmdletWebResponseException", ErrorCategory.InvalidOperation, request);
                                     string detailMsg = string.Empty;
-                                    StreamReader reader = null;
+                                    using StreamReader reader = new(StreamHelper.GetResponseStream(response));
```

Comment:
I believe it should be in try-catch. Please revert.

Reply:
reverted